### ISSUE TITLE
[Main Branch] Alternative for Enhanced Queues Tab?

### ISSUE THREAD

**Issue by OskarEichler** [ixti/sidekiq-throttled#144]
@ixti We saw that the new main branch got rid of the Enhanced Queues Tab. We really liked the simplicity of the enhanced UI to easily be able to pause and resume queues. Is there an alternative to get back this functionality?

**Comment by OskarEichler** [comment]
Or alternative question: What was the reason to remove it / is there any possibility to bring it back?

It's definitely a super handy feature that we use regularly, and  we will need to find an alternative before upgrading.

**Comment by ixti** [comment]
The reason was a combination of my misjudgement of how useful this feature was and desire to simplify the code. I will bring back ability to pause queues (and UI) this week.

**Comment by OskarEichler** [comment]
Amazing thanks so much @ixti 

Highly appreciate it!! 🙌🏼

**Comment by ixti** [comment]
I've started work on bringing it back (as a separate gem that will be integrated into this one) but haven't finished it yet. Hope to finish it this week.

The idea is that it will become completely optional, and possible to have queues pauser without throttling (when needed)

**Comment by OskarEichler** [comment]
Very exciting!! Thank you @ixti 🙏🏼

**Comment by ixti** [comment]
Still have some work to do to close this, but with #145 merged in, you can use (still in alpha, until I will write specs for all the parts) [sidekiq-pauzer](https://gitlab.com/ixti/sidekiq-pauzer) gem:

* Add `gem "sidekiq-pauzer", ">= 1.0.0.alpha", "< 2"` to your `Gemfile`
* Add `require "sidekiq/pauzer/web"` in addition to where you have `require "sidekiq/web"`

**Comment by OskarEichler** [comment]
@ixti Amazing thank you! Just tested it locally and all running well! Gonna get it deployed on production this week and report back! 🤘🏼

**Comment by ixti** [comment]
I've released [sidekiq-pauzer-1.0.0](https://rubygems.org/gems/sidekiq-pauzer/versions/1.0.0). Will work on better integration with sidekiq-throttled over the next couple of weeks. Planning on requiring explicit enabling API, something like:

``` ruby
Sidekiq::Throttled.configure do |c|
  c.use :pauzer
end
```

As of now, the integration is implicit, just add `gem "sidekiq-pauzer", "~> 1.0.0"` in your Gemfile. To have pause/resume buttons in the control panel UI:

``` ruby
require "sidekiq/web"
require "sidekiq/pauzer/web"
```

**Comment by ixti** [comment]
I've releazed sidekiq-throttled v1.0.0.

As it was said earlier, it will not have the queue pausing feature. In order to bring it back you should add `sidekiq-pauzer` to your gems:

``` ruby
gem "sidekiq-throttled", "~> 1.0.0"
gem "sidekiq-pauzer", "~> 4.0.0"
```

Then configure both:

``` ruby
# Main API
require "sidekiq"
require "sidekiq/throttled"
require "sidekiq/pauzer"

# Web UI
require "sidekiq/web"
require "sidekiq/throttled/web"
require "sidekiq/pauzer/web"
```
